docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -93,12 +93,14 @@ done
 
 ```json
 {
-  "id": "fed-node",
-  "kind": "Minion",
-  "apiVersion": "v1beta1",
-  "labels": {
-    "name": "fed-node-label"
-  }
+    "apiVersion": "v1beta3",
+    "kind": "Node",
+    "metadata": {
+        "name": "fed-node"
+    },
+    "spec": {
+        "externalID": "fed-node"
```

Comment:
oh oh oh, I see what you are saying,  not "do you need this at all" but "do you need externalID"

no, there were a couple of (broken) versions where it was needed, but you are right, it isn't needed now.  I'll pull it back out.